tabconf/2021/schnorr-signatures-jimmy-song-tabconf-2021.md

```diff
@@ -0,0 +1,74 @@
+
+## OBSERVATIONS
+
+So here are some observations. Schnorr uses hash functions, which makes it a lot faster, okay? And that's the key, right? Well, that's one of the keys, is that for a full node, when you're running Schnorr, you don't have to do all of this division, which is very CPU intensive. Instead, you can use hash functions, which are a lot faster. The other thing is that Schnorr uses a commit to an elliptic curve point, instead of just the x-coordinate. That means that we can do aggregation and batching. And that means really that you can have many signatures combined into one, and many pub keys combined into one, and on-chain you can't tell the difference. Ultimately, that means that with Schnorr, we're going to get some of these features in wallets, right? Like this ability to have sort of like multi-sig without anyone on-chain knowing about it. And that's what this taproot upgrade sort of allows, is the security around how you're actually spending things. And pretty much if you do two or three multi-sig, what you end up on taproot is three branches of the tap, the mast tree, Merkleized Abstract Syntax Tree, that are all two of two. And they all look like single keys. And as far as anyone on-chain is looking at it, they can't tell if it's one or seven or 20, gives you a lot more privacy, right? And this ability to save some blockchain space, because you only really ever have one signature. I mean, it's possible you can have more, and you can certainly write scripts that are more. But ultimately, if you're designing it correctly, you should be able to get away with one signature that's aggregated rather than multiple separate signatures that you have in ECDSA.
+
+## Q&A
```

Comment:
This section needs to be in Q and A format rather than one long paragraph

Reply:
Cool!!! On it!!